src/query/api/v1/handler/namespace/ready_test.go

```diff
@@ -70,7 +70,7 @@ func TestNamespaceReadyHandler(t *testing.T) {
 		"name": "testNamespace",
 	}
 
-	req := httptest.NewRequest("POST", "/namespace/mark_ready",
+	req := httptest.NewRequest("POST", "/namespace/ready",
```

Comment:
These URLs aren't actually used by the test, but figured I'd correct them anyway.